packages/okta-react-native/index.js

```diff
@@ -0,0 +1,94 @@
+import { NativeModules, Platform } from 'react-native';
+import {
+	assertIssuer,
+  assertClientId,
+	assertRedirectUri
```

Comment:
Watch out for mixing tabs and spaces (as shown here, it can look fine in one editor and get crazy in another).  JS community has largely standardized on 2 spaces for indent and you can set your editor to translate a TAB into the 2 spaces.